### ISSUE TITLE
Site stats

### ISSUE THREAD

**Issue by alexfernandez** [fastify/website#165]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the feature has not already been requested

### 🚀 Feature Proposal

Do you have stats for the site [fastify.dev](https://fastify.dev/)? I was in the same spot for my GitHub pages site so I built [LibreCounter](https://librecounter.org/), a free, open site stats project that collects stats without cookies or tracking of any kind.

Only aggregates are stored for each site, page and day combination, so GDPR compliant (no personal data stored like IP addresses or user agents). Also no cookie warning necessary. Stats are public and available for everyone.

If you are interested I can implement a pull request, it's quite simple to integrate. No need to display the LibreCounter logo on the web pages, just a silent "pixel" of old. Since I have the site running anyway it's no additional cost for me, and also you can inspect the code (or even self-host) if needed.

You can also display an [old fashioned counter](https://librecounter.org/oldStyle.svg) or we can work out a custom solution if you want.

Let me know!

### Motivation

I am guessing you don't have stats for the site [fastify.dev](https://fastify.dev/), at least I don't know how to get them from GitHub pages. It can be useful to know how many visits you get every day, what pages are the most visited, where the visitors come from and so on. But having a cookie warning is a bummer, and also having to deal with the GDPR and the multiple legislations around the world. LibreCounter should provide an acceptable compromise with no tracking and no cookies.

[LibreCounter](https://github.com/alexfernandez/librecounter) uses Fastify by the way, so thanks for that! :pray: 

### Example

Stats would be visible at [librecounter.org/fastify.dev/show](https://librecounter.org/fastify.dev/show):

![image](https://github.com/fastify/website/assets/876570/1de33548-e2b2-4a9b-aa15-8193fa5c838f)

**Comment by alexfernandez** [comment]
I went ahead and implemented the linked pull request: https://github.com/fastify/website/pull/167. Please let me know if it's appropriate. Thanks!

**Comment by Eomm** [comment]
That's correct: we don't have stats, but we are migrating our infra under the OpenJS Foundation - so we may consider to add this feature after this move

**Comment by alexfernandez** [comment]
OK, understood! If you want stats right here and now then let me know :smiley:

**Comment by alexfernandez** [comment]
Help was not appreciated, closing. Let me know if it's needed at any point.
